migrationtools/databases.py

```diff
@@ -279,7 +279,7 @@ def update_identity(self, uid, identity_data):
         return self.execute(statement)
 
     def get_all_identities(self):
-        statement = select([self.identity]).order_by(self.identity.c.created_at)
+        statement = select([self.identity]).order_by(self.identity.c.created_at).limit(10)
```

Comment:
That was there for testing, it was removed in commit https://github.com/praekeltfoundation/momconnect-migration-tools/pull/13/commits/5fbe9389b4b7e013d396146755093c55f388d02f